### PR TITLE
changes according to #324 - changes on the default schedule for friday

### PR DESCRIPTION
**Pull request by doritrieur**

### CODE REVIEW

**Comment by eran1205** [comment]
Hi Dorit,

Please make changes also to the Seed script in the db-seed folder.

File: src/modules/graphql/resolvers/class/services/default-class-schedules.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ const indx22: TimeSlotDbModel = {
 const indx23: TimeSlotDbModel = {
   index: '8_5',
   hours: '12:45 - 13:15',
-  lesson: lunchLesson,
+  lesson: dismissLesson,
```
Comment (eran1205) [inline src/modules/graphql/resolvers/class/services/default-class-schedules.ts]: Where do you refer to "dismissLesson"?

Comment (doritrieur) [reply src/modules/graphql/resolvers/class/services/default-class-schedules.ts]: @eran1205 
this is a constant on the default-schedule-lessons.ts that refer to פיזור תלמידים
the bug requested to replace lunch with dismissLesson on that schedule slot.

I have updated the dbSeed.,

**Comment by eran1205** [comment]
Hi Dorit,

There are conflicts. Please take care. Thanks.

**Comment by doritrieur** [comment]
formatting changes. closes and opens a new one.

**Pull request closed without merging**